build.gradle

```diff
@@ -210,6 +210,27 @@ processTestResources {
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
+def createSymlinks(archivePath, symlinkLocation) {
+    exec {
+        commandLine 'ln', '-fs', archivePath, symlinkLocation
+        ignoreExitValue = false
+    }
+}
+
+// Suffix is what will be added to the symlink
+def createAllSymlinks(destinationDir, archivePath, suffix) {
```

Comment:
This method seems to be doing something very specific as it has some hardcoded file name bits in it (e.g. "/gatk???", "/hellbender-???" , ".jar" ...)

I think the name should be more specific then. Something like "createGatkJarSymlinks".